.github/workflows/ElastixGitHubActions.yml

```diff
@@ -18,24 +18,40 @@ jobs:
           - os: ubuntu-22.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
+            libtorch_url: https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-2.6.0%2Bcpu.zip
             itk-git-tag: "v5.4.1"
             cmake-build-type: "Release"
             ANNLib: "libANNlib-5.2.so"
             ANNLib2: "libANNlib-5.2.so.1"
+            TORCH_LIBS:
+              - libtorch/libtorch/lib/libc10.so
+              - libtorch/libtorch/lib/libgomp-98b21ff3.so.1
+              - libtorch/libtorch/lib/libtorch.so
+              - libtorch/libtorch/lib/libtorch_cpu.so
           - os: windows-2022
             c-compiler: "cl.exe"
             cxx-compiler: "cl.exe"
+            libtorch_url: https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-2.6.0%2Bcpu.zip
```

Comment:
Note: While this zip file ("libtorch-win-shared-with-deps-2.6.0+cpu.zip") works just fine at the CI, there's a bit of a problem here when using Visual Studio on Windows, locally. It does not have the Debug binaries, so it does not allow switching between the Debug and Release configuration easily. (Visual Studio projects usually have the Debug and Release configuration together in one project file.) Fortunately, we are not the only ones with this problem: [TorchConfig.cmake and MS Visual debug / release projects at once](https://discuss.pytorch.org/t/torchconfig-cmake-and-ms-visual-debug-release-projects-at-once/182180), Sébastien Maraux, Jun 15, 2023.

I don't have a suggestion right now, it's just something to keep in mind 🤷